### ISSUE TITLE
Tellstick sensors actual "last updated time stamp" not reflected in HA

### ISSUE THREAD

**Issue by BoneheadFraggle** [home-assistant/hassio-addons#1535]
The actual last_updated time stamp of a Tellstick sensor, the time when the last signal from the sensor was received, is not reflected in the sensor in Home Assistant. It seems like Home Assistant's "last_updated" is always the same as "last_changed".

**Example:**

The Tellstick add-on log shows:
"fineoffset          	temperature         	216  	-24.2°	        	                    	                    	2020-08-10 19:33:40"

..where "2020-08-10 19:33:40" is when the last signal from the sensor was received.

At the same time, {{ states.sensor.tellstick_id_248_temperature.last_changed }} shows "2020-08-10 17:20:49.056231+00:00" (with 2 hours compensation for UTC / GMT + 2). Almost 13 minutes difference.

This is not an issue for sensors that change their value often. 
But when you have a thermometer sensor in a temperature controlled environment, or an aquarium where there should be very few changes in the temperature, you never know if you can trust the value.

This also means that is not possible to have automations trigger on a sensor that is not updated for a long time.

Basically, there is no automatic way to know if:
1. The temperature is correct, the sensor is updated 10 seconds ago and working well
or
2. The temperature is much to low, but the sensor hasn't updated the value due to an error.

Suggestion: 
Trigger Home Assistant's "last_updated" to be changed whenever a signal is received - even if the sensor's new value is the same as the current value,
